### PR TITLE
Issue 750

### PR DESCRIPTION
**Pull request by nanounanue**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,42 +1,118 @@
...
+RUN apk del build-runtime \
+    && rm -rf /var/cache/apk/*
+
+RUN echo 'export PS1="\[$(tput setaf 4)$(tput bold)[\]triage@$(tput setaf 6)master$(tput setaf 4)$:\\w]#\[$(tput sgr0) \]"' > /home/triage/.bashrc
 
 ENTRYPOINT [ "triage" ]
+
+FROM from-master AS dirtyduck
+
+LABEL triage.version="stable"
```
Comment (shaycrk) [inline docker/Dockerfile:88]: Isn't the triage version at this stage still from master -- it doesn't seem like it gets swapped out to the pypi version until the next stage of the build below, right?

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,42 +1,118 @@
...
+ENTRYPOINT [ "bash" ]
+
+FROM development AS production
+
+LABEL triage.version="stable"
+
+USER root
+
+RUN pip uninstall triage ipython jupyter \
+    && pip install triage
```
Comment (shaycrk) [inline docker/Dockerfile:107]: If the stable version of triage requires an older version of a module than is used by the version at master, will pip handle that dependency correctly here, downgrading the required module?

**Comment by shaycrk** [comment]
I'm a little worried about the potential for divergence between the docker requirements.txt and the main one (e.g., people updating one but forgetting the other). Is there a way to unify these?

**Comment by nanounanue** [comment]
It is handled correctly.

Already tested


On Fri, Jun 5, 2020, at 09:13, Kit Rodolfa wrote:
> 

> *@shaycrk* commented on this pull request.

> 
> In docker/Dockerfile <https://github.com/dssg/triage/pull/757#discussion_r435950462>:

> > +RUN pip install ipython jupyter
+
+USER ${USERNAME}
+
+RUN echo 'export PS1="\[$(tput setaf 4)$(tput bold)[\]\u@$(tput setaf 6)$(tput smul)dirtyduck$(tput rmul)$(tput setaf 4)$:\\w]#\[$(tput sgr0) \]"' > /home/triage/.bashrc
+
+ENTRYPOINT [ "bash" ]
+
+FROM development AS production
+
+LABEL triage.version="stable"
+
+USER root
+
+RUN pip uninstall triage ipython jupyter \
+    && pip install triage

**Comment by nanounanue** [comment]
Me too!

Working on it, I will create an issue for it

File: docker/Dockerfile (hunk 2 above)
Comment (nanounanue) [reply docker/Dockerfile:107]: It will

File: docker/Dockerfile (hunk 1 above)
Comment (nanounanue) [reply docker/Dockerfile:88]: `dirtyduck` will use the version form master

Comment (nanounanue) [reply docker/Dockerfile]: Just added a different `LABEL` in order to avoid confussion

**Pull request merged**